infra/src/main/kotlin/fi/fta/geoviite/infra/linking/PublicationDao.kt

```diff
@@ -67,7 +74,7 @@ class PublicationDao(jdbcTemplateParam: NamedParameterJdbcTemplate?) : DaoBase(j
                 trackNumberId = rs.getIntId("track_number_id"),
                 draftChangeTime = rs.getInstant("change_time"),
                 userName = UserName(rs.getString("change_user")),
-                operation = null,
+                operation = rs.getEnumOrNull<Operation>("operation") ?: Operation.MODIFY,
```

Comment:
Pituusmittauslinjan operaatio päätellään ratanumerosta. Jos ratanumeroa ei ole julkaisussa --> tulkitaan geometriamuutokseksi.

Reply:
En ole varma sainko ihan käännettyä aivojani oikeaan vinkkeliin näin heti maanantaiaamun alkuun että saisin selville onko tässä jotain vikaa vai oliko tuo vain huomio, mutta mietittiin tätä viime viikolla yhdessä Tiandun ja Karin kanssa jossain kohtaa. Me tultiin siihen tulokseen että MODIFY on ihan hyvä "default" mikäli tarkempaa arvoa ei saada laskettua. Eli jos tuo uutena tullut laskentalogiikka ei peitä mitään muuta, oikeampaa arvoa alleen, niin siten tuo palauttaa vähintäänkin speksatun arvon